app/workers/reduce_worker.rb

```diff
@@ -7,11 +7,16 @@ class ReduceWorker
     (count ** 8) + 15 + (rand(30) * count + 1)
   end
 
-  def perform(workflow_id, subject_id, user_id)
+  def perform(workflow_id, subject_id, user_id, extract_ids = [])
```

Comment:
Do we want to keep `extract_ids` out of the list considered by Sidekiq::Unique?

Reply:
that makes sense to me but i'm not quite sure how to do it

Reply:
note to self: https://github.com/mhenrixon/sidekiq-unique-jobs#finer-control-over-uniqueness

Reply:
done!